tests/agent/jobs/test_job_processing.py

```diff
@@ -0,0 +1,96 @@
+import os
+import time
+
+import pytest
+import requests
+
+org_id = os.getenv("GX_CLOUD_ORGANIZATION_ID")
+token = os.getenv("GX_CLOUD_ACCESS_TOKEN")
+gql_url = url = f"http://localhost:5000/organizations/{org_id}/graphql"
```

Comment:
```suggestion
import time

import pytest
import requests

from great_expectations_cloud.agent.config import GxAgentEnvVars

AGENT_ENV_VARS = GxAgentEnvVars()
GQL_URL = f"http://localhost:5000/organizations/{AGENT_ENV_VARS.gx_cloud_organization_id}/graphql"
```

Reply:
☝️ Having said that.
This is the kind of that fits nicely in a pytest fixture.
https://docs.pytest.org/en/7.4.x/explanation/fixtures.html#about-fixtures

```python
from great_expectations_cloud.agent.config import GxAgentEnvVars
AGENT_ENV_VARS = GxAgentEnvVars()
GQL_URL = f"http://localhost:5000/organizations/{AGENT_ENV_VARS.gx_cloud_organization_id}/graphql"

@pytest.fixture(scope="session")
def local_gql_url() -> str:
    agent_vars = GxAgentEnvVars()
    gql_url = f"http://localhost:5000/organizations/{agent_vars.gx_cloud_organization_id}/graphql"
    return gql_url

@pytest.fixture(scope="session")
def wait_for_docker_compose(local_gql_url: str):
    # pretest setup
    get_agent_status_body = """
        query agent_status {
            agentStatus {
            active
            __typename
            }
        }
    """
    response = requests.post(
        url=local_gql_url,
        json={"query": get_agent_status_body},
        headers={
            "Content-Type": "application/json",
        ...
```